test/fixtures/o-test/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "ecmaFeatures": {
+    "modules": true
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "rules": {
+    "eqeqeq": 2,
+    "guard-for-in": 2,
+    "no-extend-native": 2,
+    "wrap-iife": 2,
+    "new-cap": 2,
+    "no-caller": 2,
+    "no-irregular-whitespace": 1,
+    "no-trailing-spaces": 2,
+    "no-multi-str": 0,
+    "dot-notation": 0,
+    "strict": [2, "function"],
+    "quotes": [1, "single"],
```

Comment:
Not sure if it's helpful, but if you want to keep the warnings but not see them all the time, you can run ESLint with `--quiet` which will prevent display of warnings.  

Also, if you're getting a lot of errors/warnings, I made [eslint-nibble](https://github.com/IanVS/eslint-nibble) to help sort through them and make it less overwhelming.